kafka/consumer.go

```diff
@@ -47,13 +47,23 @@ type Consumer struct {
 	rebalanceCb        RebalanceCb
 	appReassigned      bool
 	appRebalanceEnable bool // SerializerConfig setting
+
+	isClosed bool
 }
 
 // Strings returns a human readable name for a Consumer instance
 func (c *Consumer) String() string {
 	return c.handle.String()
 }
 
+// verifyConsumer verifies if Producer can be used
+func (c *Consumer) verifyConsumer() bool {
```

Comment:
Since these functions are all the same across client types, I recommend putting this on the handle type instead and rename it to verifyClient